cobc/cobc.c

```diff
@@ -5558,6 +5557,35 @@ print_fields_in_section (struct cb_field *first_field_in_section)
 	return found;
 }
 
+/* add a "receiving" entry for a given field reference
+   and increment used counter */
+void
+cobc_xref_set_receiving (const cb_tree target_ext)
+{
+	cb_tree	target = target_ext;
+	struct cb_field		*target_fld;
+	int				xref_line;
+
+	if (CB_CAST_P (target)) {
+		target = CB_CAST (target)->val;
+	}
+	if (!CB_REF_OR_FIELD_P (target)) {
+		return;
+	}
+	target_fld = CB_FIELD_PTR (target);
+	target_fld->count++;
+#ifdef COB_INTERNAL_XREF
```

Comment:
As is (set in tree.h) this doesn't make any sense - maybe `#if COB_INTERNAL_XREF = 1`?
But that should be fixed in and (or only) merged from 3.x.

Reply:
I fail to see the problem.

`COB_INTERNAL_XREF` is defined in `tree.h` iff `COB_EXTERNAL_XREF` is not defined - that seems to make sense ?

Reply:
Ah, yes, that's fine.

We _may_ consider to drop support for internal use of cobxref in GC4 completely, it was likely never used that way in GC4 and I'm not sure if there are really much users of that feature in general.
What's your take on this?

Reply:
I guess it's better to avoid relying on an external tool for this ; let's drop it.
Maybe I should create a GitHub issue with a list of tasks to perform once all the patches are merged ; this would be an entry in that list.